README.md

```diff
@@ -314,11 +263,9 @@ When you are done, you can delete all the Azure resources created with this temp
 azd down
 ```
 
-## AI chatbot integration
+## Developer guidelines
 
-The Contoso Real Estate application can be integrated with an AI support chatbot built using [Azure OpenAI](https://azure.microsoft.com/products/ai-services/openai-service).
-
-For more details about enabling the chatbot integration, see [this section](./docs/chatbot-integration.md).
+The project has a standalone [Developer Guide](./packages/docs/website/README.md) defined under `packages/docs` and implemented as an interactive website using the [Docusaurus](https://docusaurus.io) platform. To learn more about it, [go here](./packages/docs/website/developer-guidelines.md).
```

Comment:
Good addition. We'll try to get this moved into Training modules in the near future so the content is on a 1P resource and more discoverable.